### PR TITLE
Translate English sentence to Japanese in jp.json

### PR DESCRIPTION
**Pull request by YoshikazuKashiwazaki**

### CODE REVIEW

File: src/assets/i18n/jp.json [hunk 1]
```diff
@@ -61,29 +61,29 @@
...
-    "error": "Upload error. Please try again later."
+    "confirmShare": "共有可能なリンクを作成しますか?",
+    "confirmShareDesc": "(クラウド上にアップロードされたあなたのデータは一時的に保存されています.. データは72時間後に削除されます.)",
+    "uploading": "データをアップロードして共有可能なリンクを作成します.",
+    "copyLink": "あなたのデータを共有するにはこちらのリンクをコピーしてください.",
+    "shareToWhom": "あなたはデータをどなたと共有したいですか?",
+    "cancel": "キャンセル",
+    "confirm": "確認",
+    "skip": "省略",
+    "placeholder": "例）e.g. 名無しの権兵衛",
```
Comment (dixonsiu) [inline src/assets/i18n/jp.json:85]:
```suggestion
    "placeholder": "例）名無しの権兵衛",
```

File: src/assets/i18n/jp.json [hunk 2]
```diff
@@ -61,29 +61,29 @@
...
-    "shareToWhom": "Who would you like to share your data with?",
-    "cancel": "Cancel",
-    "confirm": "Confirm",
-    "skip": "Skip",
-    "placeholder": "e.g. John Doe",
-    "success": "Data shared",
-    "error": "Upload error. Please try again later."
+    "confirmShare": "共有可能なリンクを作成しますか?",
+    "confirmShareDesc": "(クラウド上にアップロードされたあなたのデータは一時的に保存されています.. データは72時間後に削除されます.)",
+    "uploading": "データをアップロードして共有可能なリンクを作成します.",
```
[the remark this answers is not in the document]
Comment (dixonsiu) [reply src/assets/i18n/jp.json:79]: Present continuous, therefore, I rearrange the sentence. It is kind of like those message for status. e.g. "Uploading..."

File: src/assets/i18n/jp.json [hunk 3]
```diff
@@ -61,29 +61,29 @@
     "soreThroat": "喉の痛み",
     "submit": "送信",
     "clear": "解除",
-    "recorded": "記録が保存されました",
+    "recorded": "記録が保存されました.",
```
Comment (dixonsiu) [inline src/assets/i18n/jp.json:64]:
```suggestion
    "recorded": "記録が保存されました。",
```

File: src/assets/i18n/jp.json [hunk 4]
```diff
@@ -61,29 +61,29 @@
...
     "ok": "確定",
-    "saving": "Adding new data and verifiable information..."
+    "saving": "新たなデータまたは検証可能な情報の追加"
```
Comment (dixonsiu) [inline src/assets/i18n/jp.json:66]:
```suggestion
    "saving": "新たなデータまたは検証可能な情報を追加しています..."
```

**Comment by dixonsiu** [comment]
@bafu 

Can you approve and merge this pull request?

**Comment by bafu** [comment]
@YoshikazuKashiwazaki @dixonsiu Thanks for the pull request!

**Comment by bafu** [comment]
@YoshikazuKashiwazaki Can you help change the target branch from `master` to `develop`? Thanks!

**Comment by YoshikazuKashiwazaki** [comment]
@bafu Thank you for the merging and information on the target branch. Sure, I wll change the target branch from master to develop next time I make a pull request not to bother you.

**Pull request merged**